packages/webpack/test/e2e-scuttle.spec.js

```diff
@@ -0,0 +1,87 @@
+const test = require('ava')
+const path = require('node:path')
+// eslint-disable-next-line ava/no-import-test-files
+const { scaffold, runScriptWithSES } = require('./scaffold.js')
+const {makeConfig} = require('./fixtures/main/webpack.config.js')
+
+const err = (intrinsic) =>
+  'LavaMoat - property "' +
+  intrinsic +
+  '" of globalThis is inaccessible under ' +
+  'scuttling mode. To learn more visit https://github.com/LavaMoat/LavaMoat/pull/360.'
+
+async function scuttle(t, scuttleGlobalThis, globals) {
+  const webpackConfigDefault = makeConfig({
+    scuttleGlobalThis,
+    generatePolicy: true,
+    diagnosticsVerbosity: 1,
+    policyLocation: path.resolve(__dirname, 'fixtures/main/policy-scuttling'),
+  })
+  const webpackConfig = {
+    ...webpackConfigDefault,
+    entry: {
+      app: './simple.js',
+    },
+  }
+  await t.notThrowsAsync(async () => {
+    t.context.build = await scaffold(webpackConfig)
+    t.context.bundle = t.context.build.snapshot['/dist/app.js']
```

Comment:
are these paths guaranteed to be POSIX?

Reply:
this is a copy-paste from the rest of the code, should we maybe address all occurrences in a separate or?
also, i am not sure

Reply:
it's a memFs, so they should be all posix.